aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -2424,6 +2424,22 @@ memorydb:
   tags:
     product_categories: {}
   version: memorydb-2021-01-01
+neptune:
+  long: '&neptunelong;'
+  short: '&neptune;'
+  sort: Forecast
+  sdk_id: forecast
+  expanded:
+    long: Amazon Neptune Service
+    short: Neptune
+  blurb: is a fully managed graph database service by AWS, designed specifically for handling complex relationships and connected datasets at scale.
+  guide:
+    subtitle: User Guide
+    url: neptune/latest/userguide/intro.html
+  api_ref: neptune/latest/userguide/api.html
+  tags:
+    product_categories: {'Databases'}
+  version: neptune-2018-06-26
```

Comment:
Wrong version.

Reply:
Bedrock query states: 

"Yes, the statement is correct. As of May 2025, the latest officially supported Amazon Neptune Management API version remains `neptune-2018-06-26`.

Reply:
The AI answer is wrong.

Reply:
Took from the URL following instructions in QUIP